test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/VersionRange.java

```diff
@@ -0,0 +1,31 @@
+package org.elasticsearch.test.rest.yaml.section;
+
+import org.elasticsearch.Version;
+
+public class VersionRange {
+    private final Version lower;
+    private final Version upper;
+
+    public VersionRange(Version lower, Version upper) {
+        this.lower = lower;
+        this.upper = upper;
+    }
+
+    public Version lower() {
```

Comment:
Do we not usually use bean-style naming, i.e. `getLower` and `getUpper`?

Reply:
you are right, will fix